docs/extensions-plugins/cli/commands.mdx

```diff
@@ -75,7 +75,7 @@ pieces list apps
 ```
 
 #### List AI Models
-Lists all accessible AI models.
+Lists all accessible AI models, and open the list to change the model from in the `ask`
```

Comment:
Lists all accessible AI models and opens the list to change the model to be used in the `ask` command.